platforms/web/test-e2e/tests/content_list_test.ts

```diff
@@ -0,0 +1,37 @@
+import { testConfigs } from '@jwp/ott-testing/constants';
+
+import constants from '#utils/constants';
+
+Feature('content list').retry(Number(process.env.TEST_RETRY_COUNT) || 0);
+
+Before(async ({ I }) => {
+  I.useConfig(testConfigs.basicNoAuth);
+
+  if (await I.isMobile()) {
+    I.openMenuDrawer();
+  }
+
+  I.click('Popular');
+});
+
+Scenario('Header button navigates to content list screen', async ({ I }) => {
+  I.see('All Popular');
+  I.seeInCurrentUrl(`${constants.baseUrl}n/`);
+});
+
+Scenario('Content list screen loads', async ({ I }) => {
+  I.seeElement('div[role="grid"]');
+  I.seeElement('div[role="gridcell"]');
+  I.seeElement('div[id="layout_grid_0-0"]');
```

Comment:
Maybe we could add a `data-testid` to div[role="grid"] and then click on a first child of it? Just thinking how we could avoid using "role" in the selector.